armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -161,11 +163,7 @@ apt update
 apt upgrade -y
 
 # development
-apt install -y  git tmux qrencode bwm-ng
-
-# build c-lightning
-apt install -y  autoconf automake build-essential git libtool libgmp-dev \
-                libsqlite3-dev python python3 net-tools zlib1g-dev libsodium-dev
+apt install -y  git tmux qrencode #bwm-ng
```

Comment:
Nit: If we don't need the `bmw-ng` package, could we remove the commented-out part of this line?